### PR TITLE
refactor: Cleaned changelog if statements

### PR DESCRIPTION
**Pull request by rackstar17**
Refactored travis.yml and cleaned changelog if statements #526

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +10,9 @@ before_install:
 - git checkout -
 script:
 - COVERAGE=y npm test
-# Run changelog-lint but only on newer versions of Node (because of syntax errors)
-- if [[ ${TRAVIS_NODE_VERSION:0:1} -ge "4" ]]; then
-    npm run changelog-lint;
-  fi
+# Run changelog-lint
```
Comment (kumar303) [inline .travis.yml]: Since this comment doesn't add anything that the command itself doesn't already convey, you can remove it

Comment (rackstar17) [inline .travis.yml]: Done.

File: .travis.yml [hunk 2]
```diff
@@ -10,10 +10,8 @@ before_install:
 - git checkout -
 script:
 - COVERAGE=y npm test
-# Run changelog-lint but only on newer versions of Node (because of syntax errors)
-- if [[ ${TRAVIS_NODE_VERSION:0:1} -ge "4" ]]; then
-    npm run changelog-lint;
-  fi
+- npm run changelog-lint;
```
Comment (kumar303) [inline .travis.yml]: oops, sorry, I didn't notice the semi-colon. Can you remove that? It's not needed since there's no longer an if statement

Comment (rackstar17) [inline .travis.yml]: Done.

**Pull request merged**